.github/workflows/ci.yml

```diff
@@ -267,14 +269,16 @@ jobs:
         with:
           name: ${{ matrix.kernel }}.tar.gz
 
-      - name: Download rootfs and unpack artifacts
+      - name: Download rootfs opensbi and unpack artifacts
         run: |
               wget -qc https://github.com/c-sky/buildroot/releases/download/v1.0.11/rootfs_rv32.ext2.tar.gz
               tar -xvf rootfs_rv32.ext2.tar.gz
+              wget -qc https://github.com/T-head-Semi/zero_stage_boot/releases/download/v1.1/opensbi-32ilp32.tar.gz
```

Comment:
The name of T-Head-Semi is set to be discontinued, and in my opinion, the integrated OpenSBI within QEMU suffices.